docs/notebooks/algorithms/detect_somas.ipynb

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
Could you throw this in the tutorial section of the docs? add it [here](https://github.com/neurodata/brainlit/blob/develop/docs/tutorial.rst)